README.md

```diff
@@ -92,6 +104,14 @@ A complete NVRAM backup and restore can be performed between similar devices and
      Perform a backup before upgrading and restore it after to preserve your settings.
      You will experience some routing issues while the coordinator rebuilds its routing table.
  - CC2531 backups can only be restored to CC2531 devices running similar firmware versions.
+ - To run "zigpy_znp.tools.nvram_read" and "zigpy_znp.tools.nvram_write" you need to have Python installed on your system as well as required dependencies.
+   - If you are running a Linux or macOS based OS you need to have python and pip installed on your system. If you do not have them installed, refer to your distribution package manager to get it set up. (On Debian/Ubuntu; `sudo apt update && sudo apt-get install python3-pip` should work. `sudo pip` ... is not the optimal way of installing packages but Python package management is out of scope for this document.)
+     - Download and extract zigpy-znp using `wget -O cc2538-bsl.zip https://codeload.github.com/zigpy/zigpy-znp/zip/master && unzip zigpy-znp.zip`
+   - If you are running a Windows based OS download [Python for Windows](https://www.python.org/downloads/) and install. After installation verify Python is correctly installed by running `python -V` in the Command Prompt (cmd.exe). It should return Python and the version number. 
+     - Download the [zipped code](https://github.com/zigpy/zigpy-znp/archive/master.zip) and extract to a folder, then run the wanted "zigpy_znp.tools.nvram_read" and "zigpy_znp.tools.nvram_write" commands from the `\zigpy-znp-master\zigpy_znp\tools\` directory via the command-line/prompt.
```

Comment:
The command line tools have no path requirements and you don't need to switch folders to access them. `python -m zigpy_znp.tools.*` will work from anywhere the package is installed, or `py -m zigpy_znp.tools.*` on Windows.

Reply:
Hopefully fixed now.